.github/actions/sonarcloud/action.yml

```diff
@@ -21,7 +20,7 @@ runs:
         SONAR_TOKEN: ${{ inputs.sonar-token }}
       with:
         args: >
-          -Dsonar.coverage.exclusions=prime-router/src/test/**,prime-router/src/testIntegration/**,prime-router/src/main/kotlin/cli/tests/**,frontend-react/**/__mocks__/**,frontend-react/**/mocks/**,frontend-react/**/*.test.*
+          -Dsonar.coverage.exclusions=prime-router/src/test/**,prime-router/src/testIntegration/**,prime-router/src/main/kotlin/cli/tests/**,frontend-react/**/__mocks__/**,frontend-react/**/mocks/**,frontend-react/**/*.test.*,prime-router/src/main/java/**
```

Comment:
Why exclude the 51 java files in `prime-router/src/main/java/**` ?

Reply:
These are pretty just going to be POJO files/configuration and there is limited value in them getting scanned.  I don't feel super strongly about it though, so I'm happy to add a follow up ticket that will set up sonar to scan java files (it wasn't working off the bat which is why I excluded them)

Reply:
This only excludes them from coverage statistics.  Maybe the coverage supporting files are not generated for this new folder.  Please ticket for investigation.